ProteGO/Common/Networking/GcpClient/GcpEndpoint.swift

```diff
@@ -47,6 +53,8 @@ enum GcpEndpoint {
             return try? JSONEncoder().encode(body)
         case .confirmRegistration(let body):
             return try? JSONEncoder().encode(body)
+        case .getStatus(let body):
+            return try? JSONEncoder().encode(body)
```

Comment:
Czy użycie tutaj opcjonalności odpowiednio wskazuje co poszło nie tak? Może uzasadnione byłoby użycie rzucenia wyjątku lub asercja? Zwrócenie tu pustego Data jest uzasadnione w przypadku braku body ale w przypadku danych  użycie try? nie daje prawdziwego błędu.

Reply:
Wprowadziłem tutaj nowy endpoint zgodnie z architekturą zaproponowaną przez @jerzyKleszcz proponuje przedyskutowac to tutaj i ew. zrobić oddzielny issue i PR z poprawką tego problemu

Reply:
Tak, moim zdaniem powinniśmy propagować błędy kodowania do `GcpClienta`. Myślę, że zrobienie issue na to będzie ok.

Reply:
https://github.com/ProteGO-app/ios/issues/69